geniza/corpus/admin.py

```diff
@@ -371,15 +385,19 @@ def tabulate_queryset(self, queryset):
                     "latest_revision": doc.log_entries.last(),
                     "input_by": ";".join(
                         [
-                            str(n)
-                            for n in set(doc.log_entries.values_list("user", flat=True))
+                            user.get_full_name() or user.username
+                            for user in User.objects.filter(pk__in=all_user_ids)
```

Comment:
No need to do a separate lookup on id, you can get it directly from the log entries (you just need to not get a flat  list of user ids). Try something like this:
`[entry.user.get_full_name() or entry.user.username for entry in doc.log_entries.all()]`

I think we only want the unique list, though, so wrap it in `set()`

Reply:
One more thought — remove `script` after you've converted to a set, since pretty much every document will be modified by script at some point.